.github/workflows/release.yml

```
@@ -46,7 +46,7 @@ jobs:
           script: |
               core.setFailed('Release password didn\'t match.')
       - name: "Check if user is allowed"
-        if: ${{ github.actor != 'mattab' && github.actor != 'tsteur' && github.actor != 'mattmary' && github.actor != 'justinvelluppillai' && github.actor != 'lance-matomo' }}
+        if: ${{ github.actor != 'mattab' && github.actor != 'tsteur' && github.actor != 'mattmary' && github.actor != 'justinvelluppillai' && github.actor != 'lance-matomo' && github.actor != 'diosmosis' }}
```

When changing this, we should remove people not working at Matomo anymore
```suggestion
        if: ${{ github.actor != 'mattab' && github.actor != 'tsteur' && github.actor != 'diosmosis' }}
```